### PR TITLE
Add YouTube Music v1.20.0

### PR DESCRIPTION
**Pull request by mehdichaouch**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online youtube-music` is error-free.
- [x] `brew style --fix youtube-music` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask youtube-music` worked successfully.
- [x] `brew install --cask youtube-music` worked successfully.
- [x] `brew uninstall --cask youtube-music` worked successfully.

### CODE REVIEW

File: Casks/youtube-music.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+cask "youtube-music" do
```
Comment (UltraHDR) [inline Casks/youtube-music.rb:1]: This is an unofficial client, should it be renamed to youtube-music-th-ch ? What happens if YouTube Music makes a native app in the future?

Comment (LetrixZ) [reply Casks/youtube-music.rb:1]: It should be renamed, although there is another unnofficial client called `yt-music`.

**Comment by mehdichaouch** [comment]
@razvanazamfirei can I get some help on how to make this contribution valid.

Thank you

**Comment by razvanazamfirei** [comment]
Hi @mehdichaouch, we cannot accept it at this time because it does not pass our signature verification check that are a requirement for new submissions:

> macOS on ARM requires applications to be signed. Please contact the upstream developer to let them know they should notarize their package.

**Pull request closed without merging**